### ISSUE TITLE
[Bug] Some PWA's (web apps) pin to taskbar issue

### ISSUE THREAD

**Issue by Aosneg** [Floorp-Projects/Floorp#665]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

When I try to add some sites as PWA and pin them to taskbar, they are turn into default Floorp shortcut and work accordingly. But other sites pins without any issues.
For me, these sites are https://bookmarkos.com/ and https://calendar.google.com/calendar/u/0/r. But If I pin https://calendar.google.com/calendar/u/0/r/tasks, everything works perfectly. It's so weird that I see no correlation.
I think, some other sites could also have such problem.

### To Reproduce

Go to websites I listed above. Add them as PWA's. When it opens, try pinning it to your taskbar. Then close pinned PWA and try to reopen it.

### Expected behavior

Normal behavior of PWA's.

### Screenshots

https://github.com/Floorp-Projects/Floorp/assets/138026259/817cde4f-c75b-48cd-8129-bff1d2b1488c

### OS Info

Windows 11 23H2

### Floorp Version

11.7.1

### Additional context

As temporary fix for my problem, I opened properties of bugged PWA icon, went to the folder with all PWA's, copied the name of bugged PWA. Then, I changed  "C:\Program Files\Ablaze Floorp\floorp.exe" to  "C:\Program Files\Ablaze Floorp\floorp.exe" "-profile" "C:\Users\gener\AppData\Roaming\Floorp\Profiles\mtcje3mw.default-release" "-start-ssb" "{c73e56b8-9ced-4f06-b3eb-48492cd32128}" in properties of bugged PWA icon. I did it like it was done in other PWA's.

Sorry for my bad English.

**Comment by TheChilledBuffalo** [comment]
I think its an isolated issue. I can't seem to reproduce it. I tried installing https://bookmarkos.com/ as a PWA and I don't have any of the problems you mentioned.

**Comment by creeper-0910** [comment]
Please try again with a new profile.

**Comment by Aosneg** [comment]
> Please try again with a new profile.

Not working even in a new profiles.

https://github.com/Floorp-Projects/Floorp/assets/138026259/6b619b69-d81a-4ff2-8b35-2b54c0477132

**Comment by surapunoyousei** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/d42bba94-44d4-4d29-bc34-2a919d111284)

Hmm... Not found the bug

**Comment by surapunoyousei** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/3a7302f3-401a-4133-b6ab-713d7e955bf9)

**Comment by surapunoyousei** [comment]
If you got the bug. Please use install site as an app. It can be used on menu panel

**Comment by creeper-0910** [comment]
In Windows, this may be due to the icon cache.

**Comment by Aosneg** [comment]
> In Windows, this may be due to the icon cache.

Tried to delete it according to your advice and issue is still there.

I also asked my friend on Win10 to install Floorp and try to install BookmarkOS as PWA and he showed me in Discord that has the exact same problem.

![image](https://github.com/Floorp-Projects/Floorp/assets/138026259/f652684c-0fc5-4556-be90-152129730ee7)

After that, I installed portable version, did that "PWA ritual" and problem is still here.

I don't know. Maybe, I should try to install as PWA as much sites as possible to make it easier to figure out a pattern by which Web Apps have problem with pin to taskbar.

**Comment by creeper-0910** [comment]
Are you using a proxy of some kind?

**Comment by Aosneg** [comment]
> Are you using a proxy of some kind?

I'm using [GoodbyeDPI](https://github.com/ValdikSS/GoodbyeDPI). This thing is somehow modifying requests (idk) to bypass ISP restrictions.
I just tried turn it off and maybe this is the key! I tried to pin Google Calendar and it worked, but BookmarkOS is still don't want to pin properly. On the other hand, I can pin Google Calendar to taskbar even after re-enabling GoodbyeDPI. So, I'm not sure.
My friend, which I asked to install Floorp and to check PWA pinning is also using GoodbyeDPI, by the way. But he is busy now and I can't ask him to check it without GoodbyeDPI.

Additionally, we both are living in eastern part o Ukraine, so we are using Russian ISP which can be somewhat cursed. It often displays like Ukrainian, but also sometimes as Russian. We have a lot of blocked sites. And some EU and USA services are also blocking us or don't work properly. And it's usual that sites give me captcha to access them even if I visited the site for the first time.
